packages/tessellate-request/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "tessellate-request",
+  "version": "0.0.1",
```

Comment:
Let' start with 0.1.0 or even 0.0.0. [semantic-release](https://github.com/semantic-release/semantic-release) is going to automatically set the version based on the commit log when the build runs on Travis.

I have not figured out yet how to keep new projects at major version 0 after they have been created though, so semantic-release is likely going to set it to 1.0.0 the first time.

Also I don't think that dependent projects are going to be updated. So after Travis has run the semantic-release job, we are probably going to have to bump the dependency in the other projects.